src/screens/ContactScreen/index.js

```diff
@@ -269,7 +269,7 @@ const ContactScreen = ({navigation}) => {
 
   return (
     <SafeAreaView style={styles.container}>
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `barStyle` prop of the `StatusBar` component has been set to `'light-content'`. This change will make the status bar content light (white) which is suitable for dark backgrounds. However, please ensure that this change doesn't affect readability on screens with lighter backgrounds.
```diff
- <StatusBar translucent={false} />
+ <StatusBar translucent={false} barStyle={'light-content'} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->